odd-platform-ui/src/components/DataEntityDetails/DataEntityDetails.tsx

```diff
@@ -52,56 +53,62 @@ const DataEntityDetailsView: React.FC<DataEntityDetailsProps> = ({
   dataEntityId,
   dataEntityDetails,
   isDataset,
+  isQualityTest,
   fetchDataEntityDetails,
   dataEntityFetchingStatus,
-  dataEntityFetchingError
+  dataEntityFetchingError,
 }) => {
-  const [tabs, setTabs] = React.useState<AppTabItem[]>([
-    {
-      name: 'Overview',
-      link: dataEntityOverviewPath(dataEntityId),
-      value: 'overview',
-    },
-    {
-      name: 'Structure',
-      link: datasetStructurePath(dataEntityId),
-      hidden: true,
-      value: 'structure',
-    },
-    {
-      name: 'Lineage',
-      link: dataEntityLineagePath(dataEntityId),
-      value: 'lineage',
-    },
-    {
-      name: 'Test reports',
-      link: dataEntityTestReportPath(dataEntityId),
-      hidden: true,
-      value: 'test-reports',
-    },
-    {
-      name: 'Alerts',
-      link: dataEntityAlertsPath(dataEntityId),
-      value: 'alerts',
-    },
-  ]);
-
-  const [selectedTab, setSelectedTab] = React.useState<number>(-1);
-
-  React.useEffect(() => {
-    fetchDataEntityDetails({ dataEntityId });
-  }, [fetchDataEntityDetails, dataEntityId]);
+  const [tabs, setTabs] = React.useState<AppTabItem[]>([]);
 
   React.useEffect(() => {
     setTabs(
       tabs.map(tab => ({
         ...tab,
         hidden:
-          (tab.value === 'structure' || tab.value === 'test-reports') &&
-          !isDataset,
+          ((tab.value === 'structure' || tab.value === 'test-reports') &&
+            !isDataset) ||
+          (tab.value === 'lineage' && isQualityTest),
       }))
     );
-  }, [dataEntityDetails]);
+  }, [isDataset, dataEntityDetails]);
+
+  React.useEffect(() => {
```

Comment:
Probably it is better to combine these to useEffect blocks and set 'hidden' to each tab without additional loop, for example:
```
      ...
      {
        name: 'Structure',
        link: datasetStructurePath(dataEntityId),
        hidden: !isDataset,
        value: 'structure',
      },
            {
        name: 'Lineage',
        link: dataEntityLineagePath(dataEntityId),
        hidden: isQualityTest,
        value: 'lineage',
      },
      ...
```